src/Components/Publishing/Display/Canvas/CanvasVideo.tsx

```diff
@@ -134,11 +134,14 @@ export class CanvasVideo extends Component<CanvasVideoProps, any> {
 
   render() {
     const { isPlaying } = this.state
+    const { coverUrl, src } = this.props
 
-    const resized_cover = resize(this.props.coverUrl, {
-      width: 760,
-      isDisplayAd: true,
-    })
+    const resized_cover =
```

Comment:
👍 @eessex we should also add a check in `resize` to make sure we never send empty urls to gemini

Reply:
something as simple as

```javascript
if (!url) return null
```

Here https://github.com/artsy/reaction/blob/master/src/Utils/resizer.ts#L58

Reply:
Good call will add this